x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Create your first curation');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/curations-guide.html')
+      expect.stringContaining(`${docLinks.appSearchCurations}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchCurations)
```
No need for a template literal here